app/src/main/java/com/codandotv/streamplayerapp/navigation/NavigationGraph.kt

```diff
@@ -14,11 +14,12 @@ import androidx.navigation.compose.composable
 import com.codandotv.streamplayerapp.core_navigation.bottomnavigation.StreamPlayerBottomNavigation
 import com.codandotv.streamplayerapp.core_navigation.routes.BottomNavRoutes
 import com.codandotv.streamplayerapp.core_navigation.routes.Routes
-import com.codandotv.streamplayerapp.feature_list_streams.detail.presentation.navigation.detailStreamNavGraph
+import detail.presentation.navigation.detailStreamNavGraph
```

Comment:
**#question**

Is something wrong with this name space?

We should import something like:

```suggestion
import com.codandotv.streamplayerapp.feature_list_stream.detail.presentation.navigation.detailStreamNavGraph
```